README.md

```diff
@@ -311,6 +289,140 @@ ICP based refinement can be applied to the poses of the input pointclouds before
 | `publish_pointclouds` | If true the tsdf and esdf (if generated) is published as a pointcloud when the mesh is updated | false |
 | `intensity_colormap` | If the incoming pointcloud is an intensity (not RGB) pointcloud, such as from laser, this sets how the intensities will be mapped to a color. Valid options are `rainbow`, `inverse_rainbow`, `grayscale`, `inverse_grayscale`, `ironbow` (thermal) | `rainbow` |
 | `intensity_max_value` | Maximum value to use for the intensity mapping. Minimum value is always 0. | 100.0 |
+| `publish_traversable` | Whether to display a traversability pointcloud from an ESDF server. | false |
+| `traversability_radius` | The minimum radius at which a point is considered traversable. | 1.0 |
+
+# Using voxblox for planning
+The planners described in [Continuous-Time Trajectory Optimization for Online UAV Replanning](http://helenol.github.io/publications/iros_2016_replanning.pdf), [Safe Local Exploration for Replanning in Cluttered Unknown Environments for Micro-Aerial Vehicles](http://helenol.github.io/publications/ral_2018_local_exploration.pdf), and [Sparse 3D Topological Graphs for Micro-Aerial Vehicle Planning](https://arxiv.org/pdf/1803.04345.pdf) will be open-sourced shortly.
+
+In the mean-time, the general idea behind using voxblox for planning is to have two nodes running: one for the mapping, which ingests pointcloud data and produces both a TSDF and an ESDF, and one for planning, which subscribes to the latest ESDF layer over ROS.
+
+The planner should have a ``voxblox::EsdfServer`` as a member, and simply remap the `esdf_map_out` and `esdf_map_in` topics to match.
+
+A sample launch file is shown below:
+```xml
+<launch>
+  <arg name="robot_name" default="my_robot" />
+  <arg name="voxel_size" default="0.20" />
+  <arg name="voxels_per_side" default="16" />
+  <arg name="world_frame" default="odom" />
+  <group ns="$(arg robot_name)">
+
+    <node name="voxblox_node" pkg="voxblox_ros" type="esdf_server" output="screen" args="-alsologtostderr" clear_params="true">
+      <remap from="pointcloud" to="great_sensor/my_pointcloud"/>
+      <remap from="voxblox_node/esdf_map_out" to="esdf_map" />
+      <param name="tsdf_voxel_size" value="$(arg voxel_size)" />
+      <param name="tsdf_voxels_per_side" value="$(arg voxels_per_side)" />
+      <param name="publish_esdf_map" value="true" />
+      <param name="publish_pointclouds" value="true" />
+      <param name="esdf_max_distance_m" value="2.0" />
+      <param name="color_mode" value="color" />
```

Comment:
might make sense to make this example file a bit more minimal and remove params that are set to their defaults